modules/nw-ipfailover-configuring-vrrp-preemption.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ipfailover.adoc
+
+[id="nw-ipfailover-configuring-vrrp-preemption_{context}"]
+= Configuring VRRP preemption
+
+When a Virtual IP (VIP) on a node leaves the `fault` state by passing the check script, the VIP on the node enters the `backup` state if it has lower priority than the VIP on the node that is currently in the `master` state. However, if the VIP on the node that is leaving `fault` state has a higher priority, the preemption strategy determines its role in the cluster.
+
+The `nopreempt` strategy does not move `master` from the lower priority VIP on the host to the higher priority VIP on the host. With `preempt_delay 300`, the default, Keepalived waits the specified 300 seconds and moves `master` to the higher priority VIP on the host.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+* To specify preemption enter `oc edit dc ipf-ha-router` to edit the router deployment configuration:
```

Comment:
should be `oc edit deploy ipfailover-keepalived` to edit the ipfailover deployment configuration:

Reply:
Thank you!